styles/helpers/flex-grid.css

```diff
@@ -0,0 +1,5 @@
+.flex {
+  flex: 1;
+}
+
+@mixin flex-grid;
```

Comment:
just wondering if it should be an opt in for flex box or included in clientkit by default